### PR TITLE
KYLIN-3418 Adjust hybrid API

### PR DESCRIPTION
**Pull request by Wayne1c**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: server-base/src/main/java/org/apache/kylin/rest/job/HybridCubeCLI.java [hunk 1]
```diff
@@ -105,6 +105,18 @@ protected void execute(OptionsHelper optionsHelper) throws Exception {
         String projectName = optionsHelper.getOptionValue(OPTION_PROJECT);
         String modelName = optionsHelper.getOptionValue(OPTION_MODEL);
         String cubeNamesStr = optionsHelper.getOptionValue(OPTION_CUBES);
+
+        HybridInstance hybridInstance = hybridManager.getHybridInstance(hybridName);
+
+        if ("delete".equals(action)) {
+            if (hybridInstance == null) {
+                throw new RuntimeException("The Hybrid Cube doesn't exist, could not delete: " + hybridName);
```
Comment (shaofengshi) [inline server-base/src/main/java/org/apache/kylin/rest/job/HybridCubeCLI.java]: Please use illegal argument exception.

File: server-base/src/main/java/org/apache/kylin/rest/service/HybridService.java [hunk 2]
```diff
@@ -43,8 +44,7 @@
     @Autowired
     private AclEvaluate aclEvaluate;
 
-    public HybridInstance createHybridCube(String hybridName, String projectName, String modelName,
-            String[] cubeNames) {
+    public HybridRespone createHybridCube(String hybridName, String projectName, String modelName, String[] cubeNames) {
```
Comment (shaofengshi) [inline server-base/src/main/java/org/apache/kylin/rest/service/HybridService.java]: "HybridRespone" is from the rest layer, it shouldn't be referenced at the service layer. Here the interface should be "HybridInstance"

File: server-base/src/main/java/org/apache/kylin/rest/service/HybridService.java [hunk 3]
```diff
@@ -66,8 +66,7 @@ public HybridInstance createHybridCube(String hybridName, String projectName, St
         return getHybridInstance(hybridName);
     }
 
-    public HybridInstance updateHybridCube(String hybridName, String projectName, String modelName,
-            String[] cubeNames) {
+    public HybridRespone updateHybridCube(String hybridName, String projectName, String modelName, String[] cubeNames) {
```
Comment (shaofengshi) [inline server-base/src/main/java/org/apache/kylin/rest/service/HybridService.java]: "HybridInstance" is a better name than "HybridCube"

**Pull request merged**